components/chef-run/lib/chef-run/target_resolver.rb

```diff
@@ -20,14 +20,14 @@
 
 module ChefRun
   class TargetResolver
-    # IDeally, we'd base this on the actual current screen height
     MAX_EXPANDED_TARGETS = 24
 
     def initialize(unparsed_target, conn_options)
       @unparsed_target = unparsed_target
       @split_targets = unparsed_target.split(",")
       @conn_options = conn_options.dup
       @default_user = @conn_options.delete(:user)
+      @default_proto = @conn_options.delete(:protocol) || ChefRun::Config.connection.default_protocol
```

Comment:
This `|| ChefRun::Config.connection.default_protocol` should not be necessary since we are defaulting the connection options in the config

Reply:
Good catch -  I was thinking defaulted options didn't come back, but it's only those without any default that don't.